ONSdigital/ras-deploy
# Create respondent-home-ui pipeline

# Motivation and Context
Creating new pipeline for respondent home ui. All functional testing is
carried out in the integration tests so we don't need a set of
acceptance tests in the pipeline. We will want a smoke test at some
point at the variable stages of the pipeline.

# What has changed
Add respondent-home-ui pipeline to deploy to latest, fake pre-prod and fake prod

# How to test?
1. Go to http://respondent-home-ui-latest.apps.devtest.onsclofo.uk/
2. Enter a valid enrolment code
3. Check to see redirected

# Links
https://trello.com/c/YHoxwSCv/166-url-link-to-access-social-test-1-survey-5
